src/bindings/js/node/tests/tensor.test.js

```diff
@@ -1,113 +1,113 @@
-        expect(tensor.getShape().data).toEqual([1, 224, 224, 3]);
-    });
-    
-    test('Cannot create ov::Shape from Float32Array', () => {
-        expect(() => new ov.Tensor(ov.element.f32, Float32Array.from([1, 224, 224, 3]), data)).toThrow(/Invalid tensor argument. Cannot convert argument/);
-    });
-
-    test('Cannot create ov::Shape from ArrayBuffer', () => {
-        const shape = Int32Array.from([1, 224, 224, 3]);
-        expect(() => new ov.Tensor(ov.element.f32, shape.buffer, data)).toThrow(/Invalid tensor argument. Cannot convert argument/);
-    });
+  it('ov::Shape from an array object', () => {
+    const tensor = new ov.Tensor(ov.element.f32, [1, 3, 224, 224], data);
+    assert.deepStrictEqual(tensor.getShape().data, [1, 3, 224, 224]);
+  });
+
+  it('ov::Shape from an array object with floating point numbers', () => {
+    const tensor =
+    new ov.Tensor(ov.element.f32, [1, 3.0, 224.8, 224.4], data);
+    assert.deepStrictEqual(tensor.getShape().data, [1, 3, 224, 224]);
+  });
+
+  it('Array argument to create ov::Shape can only contain numbers', () => {
+    assert.throws(
+      () => new ov.Tensor(ov.element.f32, ['1', 3, 224, 224], data),
+      {message: 'Invalid tensor argument. '
+    + 'Passed array must contain only numbers.'});
+  });
+
+  it('ov::Shape from TypedArray -> Int32Array', () => {
+    const shp = Int32Array.from([1, 224, 224, 3]);
+    const tensor = new ov.Tensor(ov.element.f32, shp, data);
+    assert.deepStrictEqual(tensor.getShape().data, [1, 224, 224, 3]);
+  });
+
+  it('Cannot create ov::Shape from Float32Array', () => {
+    const shape = Float32Array.from([1, 224, 224, 3]);
+    assert.throws(
+      () => new ov.Tensor(ov.element.f32, shape, data),
+      /Invalid tensor argument./
+    );
+  });
+
+  it('Cannot create ov::Shape from ArrayBuffer', () => {
+    const shape = Int32Array.from([1, 224, 224, 3]);
+    assert.throws(
+      () => new ov.Tensor(ov.element.f32, shape.buffer, data),
+      /Invalid tensor argument./
+    );
+  });
 });
 
 describe('Tensor element type', () => {
-    it.each([
-        [ov.element.f32, "f32"],
-        [ov.element.f64, "f64"],
-        [ov.element.i32, "i32"],
-        [ov.element.u32, "u32"],
-    ])('compares ov.element.%p to string %p', (ov_elem, val) => {
-        expect(ov_elem).toEqual(val);
+  const ets = [ [ov.element.f32, 'f32'],
+    [ov.element.i32, 'i32'],
+    [ov.element.u32, 'u32'] ];
+
+  ets.forEach(([elemType, val]) => {
+    it(`Comparison of ov.element.${elemType} to string ${val}`, () => {
+      assert.strictEqual(elemType, val);
     });
-    it.each([
-        [ov.element.f32],
-        [ov.element.i32],
-        [ov.element.u32],
-    ])('compares ov.element %p', (ov_elem) => {
-        var tensor = new ov.Tensor(ov_elem, Int32Array.from([1, 224, 224, 3]), new Float32Array(150528));
-        expect(tensor.getPrecision()).toEqual(ov_elem);
+  });
+
+  ets.forEach(([elemType, ..._]) => {
```

Comment:
Just skip the rest of parameters
```suggestion
  ets.forEach(([elemType]) => {
```

Reply:
Array destructuring in JS never fails to surprise me